### DISCUSSION TITLE
How to get edited values in aggrid

### DISCUSSION THREAD

**Discussion by srinath-sankaragallu** [zauberzeug/nicegui#712] [Q&A]
### Description

i have tried ui.aggrid , and i made aggrid options editable cloumn as true 

- i am able to edit the cell of that column 
- able to update the cell value in ui 
- but unable to read the updated cell value back in python code.

**Comment by srinath-sankaragallu** [comment]
@zauberzeug-dev,, @falkoschindler   can you help me in this point 

while creating ui.aggrid
i have made colDefs as editable=True , 
when i try to edit the ui , the value is updating only in background am unable to fetch the new value back in my python code

**Comment by falkoschindler** [comment]
@srinath-sankaragallu Editing AG Grid cells is currently not part of the public API. But maybe we find a way to get it working, like we just did for QTable over here: https://github.com/zauberzeug/nicegui/discussions/675#discussioncomment-5509302.

To help us out, can you, please, provide a [minimum reproducible code example](https://en.wikipedia.org/wiki/Minimal_reproducible_example) of what you already tried?
